### ISSUE TITLE
Feature Request: Allow user to choose a format for the current line

### ISSUE THREAD

**Issue by Crowley1970** [notepad-plus-plus/notepad-plus-plus#14274]
In the options you can (mutually exclusively) choose whether the current line is not highlighted at all, is highlighted or if it is displayed with a boarder. In the latter case you can also define the border with.

However, the (light) color scheme is a bit unfortunate and the highlighting and the border is almost not visible at all. I would like to see an option to select a format for either of these two hightlighting options.

Highlighting the whole line:
- background color
- forground color (this would override the language style for the current line unless it is set to "default")
- bold, italic, underline
- font style and size (this would override the language style for the current line unless it is set to "default")

Highlighting via border:
- line style (solid, dashed, dotted, ...)
- line color
- line width (this one is already there)

Also it would be nice of highlighting and border could be combined.

**Comment by alankilborn** [comment]
> the (light) color scheme is a bit unfortunate and the highlighting ... is almost not visible at all

Sure, it is good to want things, but have you tried changing the background colour here to 
something more to your liking?:

![image](https://github.com/notepad-plus-plus/notepad-plus-plus/assets/77065706/ec15f13f-4ecf-43de-be64-64cbd6700864)

**Comment by Crowley1970** [comment]
Hi, thanx for the hint, I did not find that settings in the first place. While changing the background color here helps a bit it is still far from my suggestion. But I could understand if you turn it down if I was the only one who'd like that change. ;-)

**Comment by alankilborn** [comment]
I'm not the one to judge your feature request; I just suggested that setting in case you hadn't known about it.  Keep in mind that there are A LOT of open bug reports and feature requests for a limited development staff to deal with.

**Comment by Crowley1970** [comment]
Yes, I can see that there are a lot open open issues, but everyone needs a good, full backlog. ;-) Seriously, I don't mind if my suggestion is not implemented in the near future. Maybe one day it will. :-)
